test/validation/api/atomic/atomic.c

```diff
@@ -30,12 +29,17 @@ typedef struct {
 	odp_atomic_u64_t a64u_min;
 	odp_atomic_u64_t a64u_max;
 	odp_atomic_u64_t a64u_xchg;
+	odp_atomic_u64_t a64u_bit_set;
+	odp_atomic_u64_t a64u_bit_clr;
```

Comment:
These are not really needed but a32u and a64u could simply be used instead. The other fields like a32u_max are there only because they are needed at the same time as the generic a32u.

Reply:
Separate for clarity, "bit clear" scenario also requires a different initial value in `test_atomic_store()`.